### PR TITLE
12/18/2023 version bump to publish

### PR DESCRIPTION
**Pull request by Ankush-lastmile**
12/18/2023 version bump to publish

- Version bumps

PyPi
- 1.1.7

NPM
- 1.1.1

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/520).
* __->__ #520
* #524

### CODE REVIEW

**Review by rholinshead — APPROVED** [review]

**Comment by Ankush-lastmile** [comment]
going to wait for #524 gets merged before merging this

**Pull request merged**
